src/index.js

```diff
@@ -0,0 +1,48 @@
+import moreButton from './images/moreButton.png';
+import './index.css';
+
+const Todolist = document.getElementById('todolist');
+//  Create Object List of To-Do List
+const arrList = [
+  {
+    description: 'Car fix',
+    completed: true,
+    index: 0,
+  },
+  {
+    description: 'Football Time',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'Lunch Time',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Its Coding Time',
+    completed: false,
+    index: 3,
+  },
+  {
+    description: 'Zoom Meeting',
+    completed: false,
+    index: 4,
+  },
+];
```

Comment:
- [ ] **Required:** You have rendered the list excellently to the DOM, however, you are required to render them in the order of their index, such that even if the lower id was beneath it'd appear above.

**_Hint:_** Consider `Array.sort((a, b) => a.index - b.index`